node/pkg/accountant/query_test.go

```diff
@@ -0,0 +1,74 @@
+package accountant
+
+import (
+	"encoding/hex"
+	"encoding/json"
+	"reflect"
+	"testing"
+
+	"github.com/wormhole-foundation/wormhole/sdk/vaa"
+
+	cosmossdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestParseMissingObservationsResponse(t *testing.T) {
+	//TODO: Write this test once we get a sample response.
+}
+
+func TestParseBatchTransferStatusResponse(t *testing.T) {
```

Comment:
I think we should also include a test for the case where the contract doesn't know about a transfer and returns a null status.

Reply:
It was there, but I split it into separate tests to make it more obvious.